test/feature/test_mkd.py

```diff
@@ -179,6 +179,9 @@ def test_toy(self, device):
         assert_allclose(out[0, 0, :, :3], expected * .3787, atol=1e-3, rtol=1e-3)
         assert_allclose(out[0, 0, :, 3:], expected * 0, atol=1e-3, rtol=1e-3)
 
+    # TODO: review this test implementation
+    @pytest.mark.xfail(
```

Comment:
Failed on FP16?

Reply:
tests are not actually running in the ci - I manually tested just to make sure that functionalities at least don't crash.
For the CI we need to think about a proper way to handle tolerancies based on the dtype/device + exceptional cases